samples/getting-started/qrng/README.md

```diff
@@ -0,0 +1,56 @@
+---
+page_type: sample
+languages:
+- qsharp
+- python
+- csharp
+products:
+- qdk
+description: "This sample implements a quantum random number generator using Q#, a good first example to teach how to use the language."
+---
+
+# Creating random numbers with quantum computing
+
+This sample implements a quantum random number generator, a very simple application that is useful to learn how to write a first Q# code and it's integration with the host programs in C# or Python.
+
+In the Q# code (Qrng.qs) you can find the code for extracting a random bit using quantum measurements over a qubit in superposition. For more information, you can take a look at the [full tutorial](https://docs.microsoft.com/en-us/quantum/quickstarts/qrng?view=qsharp-preview).
+
+In the classical code (Host.cs for C# and host.py for Python) you will find the code to create a random integer from 0 to a maximum integer by invoking several times the Q# operation for extracting a random bit.
+
+
+## Prerequisites ##
+
+- The Microsoft [Quantum Development Kit](https://docs.microsoft.com/quantum/install-guide/).
+
+## Running the Sample ##
+
+This sample can be run in a number of different ways, depending on your preferred environment.
+
+### Python in Visual Studio Code or the Command Line ###
+
+At a terminal, run the following command:
+
+```bash
+python host.py
+```
+
+### C# in Visual Studio Code or the Command Line ###
+
+At a terminal, run the following command:
+
+```dotnetcli
+dotnet run
+```
+
+### C# in Visual Studio 2019 ###
+
+Open the `algorithms.sln` solution in Visual Studio, then right-click on **Qrng** and select "Set As StartUp Project."
```

Comment:
It doesn't look like `algorithms.sln` was modified to include the `Qrng.csproj` project?

Reply:
Oh, true, my bad. And `getting-started.sln` should be the file to modify since I'm adding this in the getting-started folder. Can you provide me some guidance on how to add the project properly, please? I've never used Visual Studio and I don't think I'm going to get this one right by myself.